module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -127,7 +130,7 @@ protected function translateList($transPrefix, $list)
     {
         $transList = [];
         foreach ($list as $current) {
-            $transList[] = $this->translateWithPrefix($transPrefix, $current);
+            $transList[] = is_string($current) ? $this->translateWithPrefix($transPrefix, $current) : $current;
```

Comment:
Maybe worth a comment here: what is `$current` when it is not a string? (e.g. something like `// $current may be a call number array instead of a string, in which case we should bypass translation`).

Reply:
I will add a comment. In the current VuFind code `pickValue` is not called with a prefix when used for callnumbers, and so `translateList` will not be called. 
I added that mostly to be safe in case there are institutions that want to translate their callnumbers with prefixes.